### PR TITLE
Feature/elasticsearch ingestion

### PR DESCRIPTION
**Pull request by mwesthelle**

### CODE REVIEW

File: witms/pipelines.py [hunk 1]
```diff
@@ -6,8 +6,22 @@
...
+class ElasticSearchPipeline:
+    def __init__(self):
+        self.items_buffer = []
+
+    def open_spider(self, spider):
+        self.client = Elasticsearch(hosts=["localhost"])
 
-class WitmsPipeline:
     def process_item(self, item, spider):
+        action = {"_index": "news-index", "_source": dict(item)}
```
Comment (D-to-the-K) [inline witms/pipelines.py]: Please review this hardcoded index name.

File: witms/pipelines.py [hunk 2]
```diff
@@ -6,8 +6,22 @@
...
 
+BUFFER_SIZE = 1000
+
+
+class ElasticSearchPipeline:
+    def __init__(self):
+        self.items_buffer = []
+
+    def open_spider(self, spider):
+        self.client = Elasticsearch(hosts=["localhost"])
```
Comment (D-to-the-K) [inline witms/pipelines.py]: Please review this hardcoded host name.

**Review by D-to-the-K — CHANGES_REQUESTED** [review]
Noticed a couple of hardcoded values. I suggest we move them to either constants or parameters (custom program argument or settings file).

File: witms/pipelines.py (hunk 1 above)
Comment (mwesthelle) [reply witms/pipelines.py]: Solved on commit 0b757fe

File: witms/pipelines.py (hunk 2 above)
Comment (mwesthelle) [reply witms/pipelines.py]: Solved on commit 0b757fe

**Review by D-to-the-K — APPROVED** [review]
Looking great

**Pull request merged**